Documentation/CRDs/ceph-client-crd.md

```diff
@@ -41,3 +41,146 @@ spec:
 ### Prerequisites
 
 This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+
+## Examples
+
+### Using a `CephClient` to create Ceph credentials
+
+To use `CephClient` to connect to a Ceph cluster:
+
+#### 1. Create the `CephClient` (`metadata.name` will be your Ceph client ID)
+
+Create and apply a `CephClient` resource YAML:
+
+`example.cephclient.yaml`
+```yaml
+---
+apiVersion: ceph.rook.io/v1
+kind: CephClient
+metadata:
+  name: sqlitevfs
+  namespace: rook-ceph
+spec:
+  caps:
+    mon: 'allow r'
+    osd: 'profile rbd pool=example'
+```
+
+The configuration above allows read access to mons, and RBD access on all OSDs via RBD targeting pool `example`. You can apply this config with `kubectl`:
+
+```console
+kubectl apply -f example.cephclient.yaml
+```
+
+#### 2. Find the generated secret for the `CephClient`
+
+Once your `CephClient` has been processed by Rook (in the rook namespace, for example `rook-ceph`), it will be updated to include your secret:
```

Comment:
All the docs assume the namespace `rook-ceph`, no need to repeat it much
```suggestion
Once your `CephClient` has been processed by Rook, it will be updated to include your secret:
```